Add style to user view profile-setting

Zama15 opened the pull request:
# Description

What did you implemented/Why did you implemented this:
Add setting-profile view
Remove email input on the first form due to duplicity

## Screenshots
VIew edited:
light mode
![image](https://github.com/BrightCoders-Institute/s10a1-proyecto-final-team-03-s10a1-proyecto-final/assets/112733483/b20461e4-ba69-4286-9fd8-2ddd479e46db)
dark mode
![image](https://github.com/BrightCoders-Institute/s10a1-proyecto-final-team-03-s10a1-proyecto-final/assets/112733483/e3404ee1-32c3-43ff-ae64-8e4140ebebc1)

Zama15 committed:
Add style to user view profile-setting

juperameza commented on app/assets/stylesheets/pages/_user_config.scss:
```diff
@@ -0,0 +1,122 @@
+  }
+
+  &__danger button {
+    @include themify {
+      background-color: darken(theme('backgroundColor-tertiary'), 0%);
+      border: 1px solid theme('colorInverted');
+      color: theme('colorInverted');
+    }
+  }
```
Isn't there a way to just add a single themify the code repeats a lot

Zama15 replied:
i did it, but i couldn't modify the themify in order to do not get error with the others files that use the themify, so i just added another mixin, maybe later in another pr i modify all the files and change the last mixin to the new one

juperameza replied:
Gud

juperameza reviewed (changes requested):
Also try to improve the UI 
Something feels off the inputs are the same - size as the buttons, and make the font of the label smaller.maybe that'll fix the issue with the inputs feeling small ans unclicklabe
Also add more font weight to the button

JuanequeX commented on app/views/devise/registrations/edit.html.erb:
```diff
@@ -1,87 +1,92 @@
+  <%= form_for(@user, url: user_path(@user), method: :patch, html: { class: "form" }) do |form| %>
+    <h2>Edit Profile</h2>
+
+    <div class="field">
+      <%= form.label :name %>
+      <%= form.text_field :name %>
+    </div>
+    
+    <div class="field">
+      <%= form.label :lastName %>
+      <%= form.text_field :lastName %>
+    </div>
+
+    <div class="field">
+      <%= form.label :birthday %>
+      <%= form.date_field :birthday %>
+    </div>
+
+    <div class="field">
+      <%= form.label :weight %>
+      <%= form.text_field :weight %>
+    </div>
+
+    <div class="field">
+      <%= form.label :height %>
+      <%= form.text_field :height %>
+    </div>
```
in each field add a `required: true` to have a end to end validation.

Zama15 replied:
done :+1:

JuanequeX commented on app/views/devise/registrations/edit.html.erb:
```diff
@@ -1,87 +1,92 @@
-  <div class="actions">
-    <%= f.submit "Update" %>
+    <div class="actions__danger">
+      <%= button_to "Delete my account", registration_path(resource_name), data: { confirm: "Are you sure?", turbo_confirm: "Are you sure?" }, method: :delete %>
```
```suggestion
      <%= button_to "Delete my account", registration_path(resource_name), data: { turbo_method: :delete, turbo_confirm: "Are you sure?" } %>
```

Zama15 replied:
done :+1:

Zama15 committed:
Modified views

The last styles was not liked so were modified, minors changes on
paddings and font size.
Add new mixin that work as the themify but inverted.

angel23v reviewed (commented):
Change the theme!!!!

Zama15 committed:
Re-made view

Zama15 committed:
Merge branch 'main' of github.com:BrightCoders-Institute/s10a1-proyecto-final-team-03-s10a1-proyecto-final into feat/#37/profile_settings_style_mobile
